src/libponyrt/pony.h

```diff
@@ -179,33 +179,6 @@ typedef struct pony_language_features_init_t
   size_t descriptor_table_size;
 } pony_language_features_init_t;
 
-/** Padding for actor types.
- *
- * 56 bytes: initial header, not including the type descriptor
- * 52/104 bytes: heap
- * 48/88 bytes: gc
- * 28/0 bytes: padding to 64 bytes, ignored
- */
-#if INTPTR_MAX == INT64_MAX
-#ifdef USE_MEMTRACK
-#  define PONY_ACTOR_PAD_SIZE 280
-#else
-#  define PONY_ACTOR_PAD_SIZE 248
-#endif
-#elif INTPTR_MAX == INT32_MAX
-#ifdef USE_MEMTRACK
-#  define PONY_ACTOR_PAD_SIZE 176
-#else
-#  define PONY_ACTOR_PAD_SIZE 160
-#endif
-#endif
-
-typedef struct pony_actor_pad_t
-{
-  pony_type_t* type;
-  char pad[PONY_ACTOR_PAD_SIZE];
-} pony_actor_pad_t;
```

Comment:
why kill off the struct? you are "using it" now in two different locations. seems reasonable to move this into actor.h and then still use it as it was before in cycle.c and actor.c.

Reply:
I plan to kill both uses of it in a future PR. Having this type exist whose memory layout is dependent on that pad size is antithetical with making the pad size be variable based on configuration settings at LLVM link/optimization time.

If you want me to, I can revert that part of the change in this PR, but even if so, I'm just going to remove it in a future PR.

What is your preference about how to proceed?

Reply:
Leave now remove later. Feels more contained and easier to understand.

Reply:
@SeanTAllen - The PR has been updated accordingly.